tensorboard/plugins/scalar/scalars_plugin_test.py

```diff
@@ -171,6 +167,28 @@ def test_scalars_with_scalars(self):
         self.assertEqual("application/json", response.headers["Content-Type"])
         self.assertEqual(self._STEPS, len(json.loads(response.get_data())))
 
+    def test_scalars_with_scalars_unspecified_run(self):
+        server = self.load_server([self._RUN_WITH_SCALARS])
+        response = server.get(
+            "/data/plugin/scalars/scalars",
+            query_string={
+                "run": None,
```

Comment:
(FWIW, this `"run": None` does nothing; okay with me to leave it in if
you want.)

Reply:
It reads slightly better as a test to me to leave it in and be explicit.